test/SmallTests/test_small.py

```diff
@@ -0,0 +1,50 @@
+import sys
+sys.path.append('../../')
+from ProjectLib import XMLtoProject
+from TimeLoopLib import TreeDynamicTimeStepping
+import unittest
+import glob
+import os
+
+filepathExampleSetups = "./Test_Setups_small/*.xml"
+xml = glob.glob(filepathExampleSetups)
+e = []
```

Comment:
better:
```suggestion
errors = []
```
try to use descriptive variable names